src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -1781,7 +1781,7 @@ export default class EthereumApi implements Api {
     const addressStateRoot = decode<EthereumRawAccount>(addressData)[2];
     trie.setContext(addressStateRoot, addressBuf, blockNum);
     const value = await trie.get(paddedPosBuff);
-    return Data.from(decode(value));
+    return Data.from(decode(value), 32);
```

Comment:
I made the same comment on @MicaiahReid 's review - wonder if we should encode the word length as a constant somewhere 🤷 ?